yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package dbzm
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+)
+
+type DbzSchema struct {
+	Type string `json:"type"`
+	Name string `json:"name"`
+	// Not decoding the rest of the fields for now.
+}
+
+type Schema struct {
+	ColName      string    `json:"name"`
+	Index        int64     `json:"index"`
+	ColDbzSchema DbzSchema `json:"schema"`
+}
+
+type TableSchema struct {
+	Columns []Schema `json:"columns"`
+}
+
+func GetTableSchema(tableName string, exportDir string) (TableSchema, error) {
+	schemaFileName := fmt.Sprintf("%s_schema.json", tableName)
+	schemaFilePath := filepath.Join(exportDir, "data", "schemas", schemaFileName)
+	schemaFile, err := os.Open(schemaFilePath)
+	if err != nil {
+		return TableSchema{}, fmt.Errorf("failed to open schema file %s: %v", schemaFilePath, err)
+	}
+	defer schemaFile.Close()
+	var tableSchema TableSchema
+	err = json.NewDecoder(schemaFile).Decode(&tableSchema)
+	if err != nil {
+		return TableSchema{}, fmt.Errorf("failed to decode schema file %s: %v", schemaFilePath, err)
+	}
+	return tableSchema, nil
+
+}
+
+func (ts *TableSchema) GetColumnSchema(columnName string) Schema {
+	for _, colSchema := range ts.Columns {
+		if colSchema.ColName == columnName {
```

Comment:
Strip quotes around `columnName` if they exist.
I think we should use case-insensitive comparison to avoid all the issues related to case sensitivity.

Reply:
here, columnName(header) and colSchema.Name(schema file) both column names are coming from debezium and they both will be with case-sensitivity. I don't think there will be any case-sensitivity issue here as such.